### ISSUE TITLE
fish completions

### ISSUE THREAD

**Issue by thernstig** [sharkdp/hyperfine#288]
It would be amazing if hyperfine could default-install completions for the awesome fish shell.

Reference: https://fishshell.com/docs/current/#where-to-put-completions

Example of other great tools that does this:

```sh
ll  /usr/share/fish/vendor_completions.d
.rw-r--r-- 4.4k root root 25 Apr 13:48 bat.fish
.rw-r--r--  51k root root 11 Mar  2:24 docker.fish
.rw-r--r--  12k root root  9 May 17:38 rg.fish
```

(since bat is another awesome sharkdp tool I suppose you already knew this 😄)

**Comment by sharkdp** [comment]
Thank you for your feedback.

Sounds good. For `hyperfine`, the `clap`-autogenerated completion files should work just fine, I think. If someone wants to implement this, using https://github.com/sharkdp/fd/pull/66 and the current version in https://github.com/sharkdp/fd/blob/master/build.rs as a guide might be useful.

**Comment by sharkdp** [comment]
closed via #290 by @four0000four

**Comment by sharkdp** [comment]
Released as [v1.10.0](https://github.com/sharkdp/hyperfine/releases/tag/v1.10.0).
